ignite/metrics/running_average.py

```diff
@@ -143,6 +143,8 @@ def attach(self, engine: Engine, name: str, _usage: Union[str, MetricUsage] = Ep
         if self.epoch_bound:
             # restart average every epoch
             engine.add_event_handler(Events.EPOCH_STARTED, self.started)
+        else:
+            engine.add_event_handler(Events.EPOCH_STARTED(once=1), self.started)
```

Comment:
Shouldn't it be just 
```suggestion
            engine.add_event_handler(Events.STARTED, self.started)
```
?